package.json

```diff
@@ -40,10 +40,13 @@
     "bottleneck": "^2.19.5",
     "collect.js": "^4.34.3",
     "eventsource": "^2.0.2",
+    "flatbuffers": "^23.5.26",
     "ip-num": "^1.4.1",
+    "js-sha3": "^0.8.0",
```

Comment:
I understand that we use an external library due to the low efficiency of the native hashing function from the nodejs crypto module?

Reply:
No, this is in browser, SHA3 is not available there so we use this low efficiency module instead.